packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -191,20 +199,49 @@ export function DevtoolsView(): React.ReactElement {
 		setQueryTimedOut(false);
 		messageRelay.postMessage(getSupportedFeaturesMessage);
 	}
+	initializeIcons();
 
 	return (
 		<ThemeContext.Provider value={{ themeInfo: selectedTheme, setTheme: setSelectedTheme }}>
 			<FluentProvider theme={selectedTheme.theme} style={{ height: "100%" }}>
 				{supportedFeatures === undefined ? (
 					queryTimedOut ? (
 						<>
-							<div>Devtools not found. Timeout exceeded.</div>
-							<Tooltip
-								content="Retry searching for Devtools"
-								relationship="description"
-							>
-								<Button onClick={retryQuery}>Search again</Button>
-							</Tooltip>
+							{showMessage && (
+								<MessageBar
+									messageBarType={MessageBarType.error}
+									isMultiline={true}
+									onDismiss={(): void => setShowMessage(false)}
+									dismissButtonAriaLabel="Close"
+									className={styles.icon}
+								>
+									DevTools was not found. Timeout exceeded.
+									<Tooltip
+										content="Retry searching for Devtools"
+										relationship="description"
+									>
+										<Button
+											style={{ marginLeft: "5px" }}
+											size="small"
+											onClick={retryQuery}
+										>
+											Search again
+										</Button>
+									</Tooltip>
+									<br />
+									<h4 style={{ fontWeight: "normal", margin: 0 }}>
+										{" "}
```

Comment:
Can we clean up the `{" "}` bits? Or do they get regenerated when running prettier?